chartfx-dataset/src/test/java/de/gsi/dataset/spi/DoubleErrorDataSetTests.java

```diff
@@ -60,14 +64,15 @@ public void defaultTests() {
         assertEquals(0, firstDataSet.getDataCount(), "check clear method");
     }
 
-    public static void checkAddPoints(final DoubleErrorDataSet dataSet, final int testCase) {
+    public static void
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'checkAddPoints(DoubleErrorDataSet, int)' has an NPath complexity of 260](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4730376)